workbench/.cypress/utils/constants.js

```diff
@@ -42,32 +42,32 @@ export const testQueries = [
   {
     title: 'Test GROUP BY',
     query: 'select count(*) from accounts group by gender;',
-    cell_idx: 5,
```

Comment:
np/question: what's the reason for the value changes here?

Reply:
id column was removed and the cell index shifted